mlflow/sklearn/utils.py

```diff
@@ -597,6 +600,12 @@ def _create_child_runs_for_parameter_search(cv_estimator, parent_run, child_tags
     """
     import pandas as pd
 
+    def first_custom_rank_column(df):
+        column_names = df.columns.values
+        for i, col_name in enumerate(column_names):
+            if "rank_test_" in col_name:
+                return column_names[i]
```

Comment:
Nit: Do we need the index `i` here? Can we do `for col_name in column_names` instead?